cli/cmd/generate.go

```diff
@@ -384,6 +396,32 @@ func addWorkloadOwnerKeyToManifest(manifst *manifest.Manifest, keyPath string) e
 	return nil
 }
 
+func addSeedshareOwnerKeyToManifest(manifst *manifest.Manifest, keyPath string) error {
+	keyData, err := os.ReadFile(keyPath)
+	if err != nil {
+		return fmt.Errorf("reading seedshare owner key: %w", err)
+	}
+	block, _ := pem.Decode(keyData)
+	if block == nil {
+		return errors.New("failed to decode PEM block")
+	}
+	var publicKey []byte
+	switch block.Type {
+	case "PUBLIC KEY":
+		publicKey = block.Bytes
```

Comment:
I'm not sure whether this is the right public key encoding. https://pkg.go.dev/crypto/x509#ParsePKCS1PublicKey says the PEM block should be `RSA PUBLIC KEY`, and https://pkg.go.dev/crypto/x509#ParsePKIXPublicKey says `PUBLIC KEY` but would be the wrong DER.

Reply:
As my code used the pkix thingy before rebase, I'm pretty sure it would work. But that code is removed anyway now.

Reply:
That's what I meant - it'd be using PKIX, but expected is PKCS1.